Makefile

```diff
@@ -311,11 +311,11 @@ deploy-operator: deploy-crd
 .PHONY: deploy-clean
 ## Deploy a CR as test
 deploy-clean:
-	@-oc delete imagestream.image.openshift.io/myapp-runtime
+	@-oc delete component.devopsconsole.openshift.io/myapp
```

Comment:
this won't execute all remaining commands if failed for the first command. maybe appending `|| true` will help here

Reply:
@-oc => carry on if failed

Reply:
yes, the `-` prefix can be used to ignore the command exit status code: https://www.gnu.org/software/make/manual/make.html#Errors-in-Recipes